base/iterators.jl

```diff
@@ -272,6 +272,8 @@ setindex!(v::Pairs, value, key) = (v.data[key] = value; v)
 get(v::Pairs, key, default) = get(v.data, key, default)
 get(f::Base.Callable, v::Pairs, key) = get(f, v.data, key)
 
+Base.map(f, A::Pairs) = collect(Base.Generator(f,A))
```

Comment:
Perhaps should be general (and in abstractdict.jl)?
```suggestion
Base.map(f, A::AbstractDict) = collect(Base.Generator(f, A))
```

Reply:
triage says yes.

Reply:
To be clear, this means that `map(f, d::Dict)` would return a vector? Then it's the same as `map(f, d)` and we have no way to apply a function and get a dictionary back like in Dictionaries.jl?